parsel/selector.py

```diff
@@ -72,6 +76,17 @@ def __getitem__(self, pos):
     def __getstate__(self):
         raise TypeError("can't pickle SelectorList objects")
 
+    def jpath(self, jpath, **kwargs):
+        """
+        Call the ``.jpath()`` method for each element in this list and return
+        their results flattened as another :class:`SelectorList`.
+
+        ``query`` is the same argument as the one in :meth:`Selector.jpath`
+
+        Requires : jmespath(https://github.com/jmespath/jmespath)
```

Comment:
This paragraph is unnecessary, since `jmespath` is being set as a dependency of Parsel already.